### PR TITLE
25 feature firebase auth persistence

### PR DESCRIPTION
**Pull request by nmWolfe**
Updated README, and fixed paths from Login, Register, Nav and Federated Sign in component. Fixed SCSS in Login to match Register page.  
Page will now keep users signed in, until we create an option to sign out.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,33 +1,53 @@
...
+
+  const auth = getAuth();
+  const navigate = useNavigate();
+
+  useEffect(
+    () => {
+      onAuthStateChanged(auth, (user) => {
+        if (user) {
+          setUser(user.uid);
+          console.log(user);
```
Comment (maxswaine) [inline src/App.tsx:28]: Remove console log

Comment (luke-welbourn) [reply src/App.tsx]: removed the console log from app.tsx

File: src/App.tsx [hunk 2]
```diff
@@ -1,35 +1,54 @@
...
 import Register from "./pages/Register/Register";
 import Events from "./pages/Events/Events";
 import About from "./pages/About/About";
 import Calendar from "./pages/Calendar/Calendar";
+import { getAuth, onAuthStateChanged } from "firebase/auth";
 import Account from "./pages/Account/Account";
 
 const App = () => {
-  const [userId, setUserId] = useState<string>("");
+  const setUserId = useState<string>("")[1];
```
Comment (maxswaine) [inline src/App.tsx:16]: Now that we have the user from the auth, we don't need this function to set a userID. It is already created by the database so we can remove this functionality. If you console log the user on sign in you can see the userid in the console already created.
Remove the setUserId state throughout the project

**Review by maxswaine — CHANGES_REQUESTED** [review]
Overall looks great. Functionality is also great. Check comments about setUserID

File: src/App.tsx (hunk 2 above)
Comment (nmWolfe) [reply src/App.tsx]: done

**Review by olirob93 — APPROVED** [review]
looks good once console log is removed

**Review by Charlie-robin — APPROVED** [review]

**Comment by Charlie-robin** [comment]
@nmWolfe @luke-welbourn If you resolve the merge conflicts we can get this in asap

**Pull request merged**